cli/dts/lib.deno.unstable.d.ts

```diff
@@ -206,12 +206,14 @@ declare namespace Deno {
   /** **UNSTABLE**: new API, yet to be vetted.
    *
    * Format an array of diagnostic items and return them as a single string in a
-   * user friendly format.
+   * user friendly format. If the diagnostic items is an empty array, it will
+   * return an empty string.
    *
    * ```ts
    * const { diagnostics } = await Deno.emit("file_with_compile_issues.ts");
    * console.table(diagnostics);  // Prints raw diagnostic data
    * console.log(Deno.formatDiagnostics(diagnostics));  // User friendly output of diagnostics
+   * console.log(Deno.formatDiagnostics([])) // ""
```

Comment:
I was confused about what `// ""` meant until I realized that's showing the output. Maybe remove this line or explain the output like the other comments? I'm not sure this needs to be explicitly called out in an example as the description above probably sufficies and I think most people would expect it to naturally return an empty string in this scenario.

Reply:
https://github.com/denoland/deno/pull/10925/commits/ef016fefeaa4e79aad4fdd95963f1e69e773e45a